### ISSUE TITLE
repetition of error lines about contrast color in the report

### ISSUE THREAD

**Issue by ptytgat** [daisy/ace#197]
ACE 1.0.1 logs an error line when the contrast between foreground and background colors doesn't reach WCAG 2 AA ; this error line is inserted  for each occurrence of the error. For some epub files, the log contains thousands of error lines, so it is difficult to read it and to locate different errors.
I suggest that ACE logs only one line for the first occurrence of this error for the same HTML context 
It would be more convenient to read the log file if ACE .

For Example for my epub :
error 1
p1chap2.xhtml#epubcfi(/4/2[chap-001]/8/6[niv1-002]/12/2)
Snippet:
<p xmlns="http://www.w3.org/1999/xhtml" class="citp-c">Face au changement<a class="indx" id="indx-33"></a>,</p>
error 2
p1chap2.xhtml#epubcfi(/4/2[chap-001]/8/6[niv1-002]/12/6)
Snippet:
<p xmlns="http://www.w3.org/1999/xhtml" class="citp-c">capable de changer…</p>
These 2 errors are inside the <p class="citp-c"> element ; the first error message would be sufficient to correct the epub( (modification of a css).

**Comment by diagram-codesprint** [comment]
I would suggest that when you add the option -verbose (if we have that) then all errors for contrast would be logged otherwise just the one as was suggested by @ptytgat

**Comment by rdeltour** [comment]
Yes, I think the feature request makes sense. Thanks for the feedback!
